src/client.cpp

```diff
@@ -173,6 +173,11 @@ CClient::CClient ( const quint16  iPortNumber,
     // start timer so that elapsed time works
     PreciseTime.start();
 
+    // set gain delay timer to single-shot and connect handler function
```

Comment:
That comment almost repeats the variable/method names and provides little added value, but I guess it's ok.. :)